web/index.html

```diff
@@ -4,7 +4,38 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <meta
+      name="description"
+      content="This website is version 2 of My-Cv which it's uses just PURE HTML5 and CSS3"
+    />
+    <link rel="stylesheet" href="css/style.css" />
     <title>My CV V2</title>
   </head>
-  <body></body>
+  <body>
+    <div class="container">
+      <h1>Bogdan Negura</h1>
+      <h2>Junior Web Developer</h2>
+      <h3>About Me</h3>
+      <section class="section">
+        <p class="p-about">
+          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Suscipit
+          nobis at expedita ipsam similique minus consequuntur incidunt, placeat
+          dolorem laboriosam. <br />
+          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Suscipit
+          nobis at expedita ipsam similique minus consequuntur incidunt, placeat
+          dolorem laboriosam.
+        </p>
+      </section>
+      <div class="center-btn">
+        <button class="download-button">Download my CV</button>
+      </div>
+      <div class="arrow-svg">
+        <img
```

Comment:
don't use the requirement folder directly, grab your files and put them in somewhere inside the `web` folder.